api/client/kill.go

```diff
@@ -17,17 +18,16 @@ func (cli *DockerCli) CmdKill(args ...string) error {
 
 	cmd.ParseFlags(args, true)
 
-	var errNames []string
+	var errs []string
 	for _, name := range cmd.Args() {
 		if err := cli.client.ContainerKill(name, *signal); err != nil {
-			fmt.Fprintf(cli.err, "%s\n", err)
```

Comment:
I think it might be best to keep this fmt.Fprintf() here so that we show the status for each container in the other in which the user provided them. Then remove the "if' down below on line 29. Then you can get rid of "errs" too.
Shouldn't this PR wait for the "set" one to be merged first?